src/api-service/__app__/onefuzzlib/azure/creds.py

```diff
@@ -99,18 +99,46 @@ def get_regions() -> List[Region]:
     return sorted([Region(x.name) for x in locations])
 
 
-@cached
-def get_graph_client() -> GraphRbacManagementClient:
-    return GraphRbacManagementClient(get_msi(), get_subscription())
+def query_microsoft_graph(
```

Comment:
Testing in progress
- [x] deploying from a linux machine 
- [ ] running integration tests